src/main/java/xbot/common/controls/actuators/XCANSparkMax.java

```diff
@@ -739,4 +774,35 @@ public abstract REVLibError setReference(double value, ControlType ctrl, int pid
     public abstract boolean getForwardLimitSwitchPressed(com.revrobotics.SparkLimitSwitch.Type switchType);
 
     public abstract boolean getReverseLimitSwitchPressed(com.revrobotics.SparkLimitSwitch.Type switchType);
+
+    public abstract REVLibError setPeriodicFramePeriod(CANSparkLowLevel.PeriodicFrame frame, int periodMs);
+
+    /**
+     * // See https://docs.revrobotics.com/sparkmax/operating-modes/control-interfaces#periodic-status-frames
```

Comment:
It might be worth adding more 1 tldr here about what status frames are, something along the lines of the tail end of this comment about pid?